index-import.cpp

```diff
@@ -152,7 +155,7 @@ static IndexUnitWriter remapUnit(const std::unique_ptr<IndexUnitReader> &reader,
       if (name != "") {
         writer.getUnitNameForOutputFile(filePath, unitName);
         if (Verbose) {
-          outs() << "DependencyUnitName: " << unitName << "\n";
+          outs << "DependencyUnitName: " << unitName.c_str() << "\n";
```

Comment:
Any reason this changed?

Reply:
`outs` was an `llvm::raw_ostream`, which supports outputting llvm types like `SmallString`. Now `outs` is a `std::ostream` that doesn't support llvm types.